yarn.lock

```diff
@@ -7645,8 +7645,8 @@ matrix-mock-request@^1.2.3:
     create-react-class "^15.6.0"
     diff-dom "^4.1.3"
     diff-match-patch "^1.0.4"
-    emojibase-data "^5.0.1"
-    emojibase-regex "^4.0.1"
+    emojibase-data "^4.0.2"
```

Comment:
This seems wrong / unrelated...?

Reply:
yarn refused to install the new dependency without doing this :(